ccmlib/dse_node.py

```diff
@@ -192,8 +192,8 @@ def dse(self, dse_options=None):
         dse = common.join_bin(self.get_install_dir(), 'bin', 'dse')
         args = [dse]
         args += dse_options
-        p = subprocess.Popen(args, env=env, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-        return handle_external_tool_process(p, args)
+        p = subprocess.Popen(args, env=env)  # Spawn an interactive shell, cannot pipe stdout/std err
```

Comment:
I don't think this comment is quite right -- it doesn't spawn a new shell, it spawns a new process, and that process inherits `std{in,out} from the parent `ccm` process. When called from a shell, this is just inherited from the shell's `std{in,out}`, so users can interact with the, e.g., `dse` process.

The comment is basically explaining the absence of `stdout` and `stdin` arguments, yeah? It took me a minute to get that. I'd rephrase as "don't redirect stdout/in so users can interact with new process" or something similar.